internal/pkg/model/template.go

```diff
@@ -8,17 +8,15 @@ import (
 type TemplateRepositoryType string
 
 const (
-	RepositoryTypeWorkingDir = `working_dir`
```

Comment:
Vyhodil som `RepositoryTypeWorkingDir`, kod je tak jednoduchsi, ... mohol som to tak upravit vdaka pridaniu `fs.SetWorkingDir`.